02.calendar/calendar.rb

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env ruby
+
+require 'date'
+require 'optparse'
+
+def generate_dates_list(year, month)
+  last_day = Date.new(year, month, -1).day
+  dates = []
+
+  (1..last_day).each do |day|
+    date = Date.new(year, month, day)
+    dates << date
+  end
+  
+  dates
```

Comment:
```suggestion
  date_from = Date.new(year, month, 1)
  date_to = Date.new(year, month, -1)
  date_from..date_to
```

rangeを使うとこれで事足りたりします。